ci/circle_build_linux.sh

```diff
@@ -22,7 +22,7 @@ conda config --add channels https://repo.continuum.io/pkgs/free
 conda config --add channels conda-forge
 
 conda create -y -q -n fletcher python=${PYTHON_VERSION} \
-    pandas pyarrow pytest pytest-cov pytest-flake8 \
+    pandas pyarrow=0.14 pytest pytest-cov pytest-flake8 \
```

Comment:
there is a problem while using `pyarrow=0.15`: https://circleci.com/gh/krivonogov/fletcher/7?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
`0.14.1`?

Reply:
conda should take last `0.14` available, so it will be `pyarrow-0.14.1` (I checked it in tests circleci output :) )